apps/api/src/app/integrations/usecases/get-integrations/get-integrations.usecase.ts

```diff
@@ -13,6 +13,7 @@ export class GetIntegrations {
       GetDecryptedIntegrationsCommand.create({
         organizationId: command.organizationId,
         userId: command.userId,
+        environmentId: command.environmentId,
```

Comment:
Why did you add the environment? As I understand it we do want all integrations, from both environments

Reply:
because the feature flag usecase requires it :)

Reply:
Oops. Sorry ;)

Reply:
@davidsoderberg mind to leave a TODO comment to remind us to remove once we sunset the pre feature functionality?